src/app/archive_blocks/archive_blocks.ml

```diff
@@ -103,6 +99,10 @@ let main ~archive_uri ~precomputed ~extensional ~success_file ~failure_file
 
 let () =
   Command.(
+    let genesis_constants = Genesis_constants.Compiled.genesis_constants in
```

Comment:
Correct, not *yet*. 

This will be coming soon, but at least now the value is being assigned in an `IO`/ top level context (here the CLI). The next step will be constructing it from the environment.